W/WCPG/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "WCPG"
+version = v"0.9.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/remi-garcia/WCPG/archive/refs/tags/v0.9.1.tar.gz", "b794f0df05d8e0a42a077dafe445ddaba550659f370ef64a40ac84bc91c8b744"),
+    FileSource("https://www.netlib.org/clapack/f2c.h", "7d323c009951dbd40201124b9302cb21daab2d98bed3d4a56b51b48958bc76ef"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/WCPG-0.9.1/
+sh autogen.sh
+./configure CFLAGS="-I${WORKSPACE}/srcdir/ -I${includedir}/" --prefix=${prefix} --build=${MACHTYPE} --host=${target}
```

Comment:
```suggestion
export CPPFLAGS="-I${WORKSPACE}/srcdir -I${includedir}"
./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
```
Note that `-I` is a flag for the preprocessor ([`CPPFLAGS`](https://www.gnu.org/software/automake/manual/html_node/Standard-Configuration-Variables.html)), not the compiler itself, even though if you give it to the compiler it'd be forwarded to the preprocessor